### ISSUE TITLE
Change Pane title when pane is Zoomed

### ISSUE THREAD

**Issue by sivaramn** [tmux/tmux#1957]
### Issue description
How  do I change the pane title to have the word "ZOOMED" blinking when I zoom a pane.  I currently have this in my set up and I'd like an indicator to show up somewhere that I had zoomed the pane.  

````
set-option -g pane-border-status top
set-option -g pane-border-format "Host:#h|Session:#S|Window:#I|Pane:#P|#{pane_current_command} "
````

IF there is a way to do it, would like to know whether something exists for synchronise-panes action too i.e. have an indicator on all panes that sync is on

### Required information

Please provide the following information:

* tmux version (`tmux -V`).
`tmux next-3.1`
* Platform (`uname -sp`)
`CYGWIN_NT-10.0 unknown `.
* $TERM inside and outside of tmux (`echo $TERM`).
`tmux-256color`

**Comment by nicm** [comment]
Use the window_zoomed_flag format.

**Comment by sivaramn** [comment]
> 
> 
> Use the window_zoomed_flag format.

thanks, I managed to get something like this done

````
set-option -g pane-border-format "Host:#h|Session:#S|Window:#I|Pane:#P|#{pane_current_command}#{?window_zoomed_flag,|#[fg=black blink bold bg=brightblue bli\
nk reverse bold]ZOOMED,} "
````

though `blink` doesn't do anything.  The words ZOOMED doesn't `blink` at all. Is there a way to get it to blink like the cursor?

**Comment by nicm** [comment]
Are you sure your TERM has blink?

**Comment by sivaramn** [comment]
> 
> 
> Are you sure your TERM has blink?

How do I check that?

**Comment by sivaramn** [comment]
> 
> 
> "tmux info|grep blink" inside tmux.

````
 tmux info|grep blink
   4: blink: (string) \033[5m
````

also
````
 infocmp.exe $TERM
#       Reconstructed via infocmp from file: /usr/share/terminfo/74/tmux-256color
tmux-256color|tmux with 256 colors,
        am, hs, km, mir, msgr, xenl,
        colors#0x100, cols#80, it#8, lines#24, pairs#0x10000,
        acsc=++\,\,--..00``aaffgghhiijjkkllmmnnooppqqrrssttuuvvwwxxyyzz{{||}}~~,
        bel=^G, blink=\E[5m, bold=\E[1m, cbt=\E[Z, civis=\E[?25l,
        clear=\E[H\E[J, cnorm=\E[34h\E[?25h, cr=\r,
        csr=\E[%i%p1%d;%p2%dr, cub=\E[%p1%dD, cub1=^H,
        ....
````

**Comment by sivaramn** [comment]
> tput blink; echo abcdef; tput sgr0

It didn't at first; on a whim, I checked the `mintty` terminal  `options-->text` menu and I found a checkbox `Allow Blinking` which i enabled.  Now, the pane title blinks.  Thanks for the help.

This may need to go into the FAQ?  If using `mintty` and its variants, users should enable various _relevant_ options in the `options` menu outside of `tmux` and then enable them explicitly in the `.tmux.conf`
